src/util/domain-icon.js

```diff
@@ -38,6 +38,9 @@ export default function domainIcon(domain, state) {
     case 'lock':
       return state && state === 'unlocked' ? 'mdi:lock-open' : 'mdi:lock';
 
+    case 'garage_door':
+      return state && state === 'open' ? 'mdi:arrow-up' : 'mdi:arrow-down';
```

Comment:
I looked at state-card-rollershutter.js and I am not sure how that would work with this line. 

return stateObj.attributes.current_position === 100;

The garage door has a state of 0 or 1?

If you can provided some guidance I can attempt to make those changes, but I am not sure what all needs changed.